pkgs/applications/science/electronics/openroad/default.nix

```
@@ -79,7 +80,12 @@ mkDerivation rec {
   ];
 
   patches = [
-    ./0001-Fix-string-formatting-in-tests.patch
+    (fetchpatch {
+      name = "openroad-fix-fmt-10.patch";
+      url = "https://github.com/The-OpenROAD-Project/OpenROAD/commit/9396f07f28e0260cd64acfc51909f6566b70e682.patch";
+      hash = "sha256-jy8K8pdhSswVz6V6otk8JAI7nndaFVMuKQ/4A3Kzwns=";
+    })
+    ./0001-Disable-failing-regression-tests.patch
     ./0002-Ignore-warning-on-stderr.patch
```

Is upstream aware of these issues?

---

No, but I can't really work on openROAD right now because or-tools is broken...

---

> No, but I can't really work on openROAD right now because or-tools is broken...

Fair enough, I understand it's not your area of responsibility, but I'd suggest something simple as:

```suggestion
    # Upstream is not aware of these failures
    ./0001-Disable-failing-regression-tests.patch
    # This is an issue we experience in the sandbox, and upstream 
    # probably wouldn't mind merging this change, but no PR was opened.
    ./0002-Ignore-warning-on-stderr.patch
```